docs/src/main/tut/contributing.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: docs
```

Comment:
I'm wondering if it makes sense to put this awesome contributing guide in the microsite main page, using `page` layout. We did it in cats in this way: http://typelevel.org/cats/contributing.html

What do you think?